### PR TITLE
add EDAM topics

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: tools/data_manipulation/xarray/xarray_select.xml [hunk 1]
```diff
@@ -261,7 +261,10 @@
              </output>
         </test>
     </tests>
-
+    <edam_topics>
```
Comment (bgruening) [inline tools/data_manipulation/xarray/xarray_select.xml:264]: could you not do this in macro?

**Comment by yvanlebras** [comment]
True.... Was hesitating but wanna not modyfying too much things there... But I think we can rethink this on a next edam / bio.tools round related to additioon of more tags ?

**Comment by bgruening** [comment]
Sure! :)

**Comment by yvanlebras** [comment]
Thank you!

**Pull request merged**